pkg/host/internal/lib/ethtool/ethtool.go

```diff
@@ -0,0 +1,51 @@
+package ethtool
+
+import (
+	"github.com/safchain/ethtool"
+)
+
+func New() EthtoolLib {
+	return &libWrapper{}
+}
+
+//go:generate ../../../../../bin/mockgen -destination mock/mock_ethtool.go -source ethtool.go
+type EthtoolLib interface {
+	// Features retrieves features of the given interface name.
+	Features(intf string) (map[string]bool, error)
+	// FeatureNames shows supported features by their name.
+	FeatureNames(intf string) (map[string]uint, error)
+	// Change requests a change in the given device's features.
+	Change(intf string, config map[string]bool) error
+}
+
+type libWrapper struct{}
+
+// Features retrieves features of the given interface name.
+func (w *libWrapper) Features(intf string) (map[string]bool, error) {
```

Comment:
let's use the same variable name in all places to specify interface name. E.g. `ifaceName`

Reply:
done